libs/ai-endpoints/tests/integration_tests/test_chat_models.py

```diff
@@ -202,9 +202,23 @@ def test_ai_endpoints_invoke(chat_model: str, mode: dict) -> None:
     assert isinstance(result.content, str)
 
 
-# todo: tests for parameters -
-#        bad: str - Bad words to avoid (cased)
-#        stop: str - Stop words (cased)
+# todo: test that stop is cased and works with multiple words
```

Comment:
from openAI API spec:
"Up to 4 sequences where the API will stop generating further tokens."